plugins/settings.gradle.kts

```diff
@@ -30,6 +30,7 @@ pluginManagement {
     includeBuild("../build-logic")
 }
 
+@Suppress("UnstableApiUsage")
```

Comment:
Why do you need this?

Reply:
The dependencyResolutionManagement block is still marked unstable, though it's been around since at least Gradle 7